python_modules/dagster/dagster/_core/telemetry.py

```diff
@@ -133,6 +144,19 @@ def get_python_version():
     return "{}.{}.{}".format(version.major, version.minor, version.micro)
 
 
+def get_is_known_ci_env():
+    # Many CI tools will use `CI` key which lets us know for sure it's a CI env
+    if os.environ.get("CI") is True:
```

Comment:
i slight worry about false positives on this but eh

Reply:
im actually pretty confident here as 1) if you set a "CI" env var that says CI = true, i think it's safe to say it's a CI. 2) most of the links that i put next to the KNOWN_CI_ENV_VAR_KEYS has `CI = true` [plus this request](https://issues.jenkins.io/browse/JENKINS-36707) in jenkins, which makes me think it's a common practice.